src/ucp/api/ucp.h

```diff
@@ -770,9 +770,16 @@ typedef struct ucp_ep_params {
     ucp_err_handling_mode_t err_mode;
 
     /**
-     * Handler to process transport level failure.
+     * Error handler callback, will be called with @ref user_data as an argument
+     * if set.
      */
-    ucp_err_handler_t       err_handler;
+    ucp_err_handler_cb_t    err_handler_cb;
+
+    /**
+     * User data associated with an endpoint. See @ref ucp_stream_poll_ep_t and
+     * @ref err_handler_cb
```

Comment:
which flag in field_mask indicates the presence of this field?

Reply:
👍 need to rename and reorder accordingly

Reply:
done, see UCP_EP_PARAM_FIELD_ERR_HANDLER_CB and UCP_EP_PARAM_FIELD_USER_DATA